client/src/Component/MainTable.js

```diff
@@ -1,11 +1,14 @@
 import React, { useState, useEffect } from "react";
 import "./MainTableStyle.css";
 import Alert from "react-bootstrap/Alert";
+import Checkbox from "./Checkbox.js";
 
-const MainTable = () => {
+const MainTable = (props) => {
 	const [clientData, setClientData] = useState([]);
 	const [error, setError] = useState(null);
 	const [isLoading, setIsLoading] = useState(false);
+	const [masterChecked, setMasterChecked] = useState(true);
```

Comment:
Since we're not implementing the select-all checkbox anymore, let's remove this unused state